lib/osf-components/addon/components/registries/schema-block-renderer/read-only/mapper/component.ts

```diff
@@ -0,0 +1,10 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import template from './template';
+
+@layout(template)
+@tagName('')
+export default class Mapper extends Component {
+}
```

Comment:
Should the read-only mapper have an assertion for `pageResponses` and the editable mapper have an assertion for `changeset`? Should those parameters be required in those? @adlius @baylee-d

Reply:
They could be required but they don't have to be... leaving them optional would make it easier to build a nice skeleton screen in the future, if we ever find ourselves with a schema but still waiting on the responses. That might be an unlikely future, though.

🤷‍♂ I'm content either way.

Reply:
(but yeah, as a general rule, we should have assertions for required params, because that's otherwise unenforced)